joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/astcreation/Defines.scala

```diff
@@ -2,6 +2,7 @@ package io.joern.gosrc2cpg.astcreation
 
 object Defines {
   val anyTypeName: String            = "ANY"
+  val tobeFilled: String             = "<TO_BE_FILLED>"
```

Comment:
Why did you choose this strategy over using `ANY` instead?

Reply:
It's one step prior to setting `ANY` if we can't resolve the TypeFullName. I am setting it `<TO_BE_FILLED>` to search for such nodes, and in the next pass `CallTypeAndSigntureLinkerPass`, I search with `<TO_BE_FILLED>` and try to resolve TypeFullName from the method cache. This will at least resolve the TypeFullName for the method calls to the methods defined within the source code. If it fails to find, then it will set it to `ANY`. 

In principle, I am yet to ascertain I am not able to resolve this Type and then set to ANY till that time it will be set to `<TO_BE_FILLED>`. This will also be helpful to troubleshoot in future, in case if there is any node that is left with this value. Then there is some use case that is not being handled in the `CallTypeAndSigntureLinkerPass`